dll/win32/kernel32/client/file/find.c

```diff
@@ -687,9 +687,10 @@ FindFirstFileExW(IN LPCWSTR lpFileName,
         /*
          * May represent many FILE_BOTH_DIR_INFORMATION
          * or many FILE_FULL_DIR_INFORMATION structures.
+         * NOTE: NtQueryDirectoryFile requires the buffer to be ULONG-aligned
          */
-        BYTE DirectoryInfo[FIND_DATA_SIZE];
-        DIR_INFORMATION DirInfo = {&DirectoryInfo};
+        DECLSPEC_ALIGN(4) BYTE DirectoryInfo[FIND_DATA_SIZE];
+        DIR_INFORMATION DirInfo = { .DirInfo = &DirectoryInfo };
```

Comment:
Does clang complain if you don't use the named member?

Reply:
No, I just added it for better readability